processing/src/main/java/io/druid/query/select/SelectQueryConfig.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Licensed to Metamarkets Group Inc. (Metamarkets) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. Metamarkets licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.druid.query.select;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class SelectQueryConfig
+{
+  public static String CTX_KEY_ENABLE_FROM_NEXT_DEFAULT = "enableFromNextDefault";
```

Comment:
This isn't a context key so it's misleading for it to start with `CTX_KEY_`. It seems like it's only used in the tests, so IMO it doesn't really need to be a constant at all. But if you want it to be, then please give it another name.